### PR TITLE
If no university login, then redirect to homepage

### LINKED ISSUE #1232: Public user login should redirect to homepage
### Descriptive summary
If a user has no roles or is only institutionally affiliated, they should be brought to the homepage, not the dashboard

### Expected behavior
non-admin users never see the admin dashboard

### PR DESCRIPTION
**Pull request by straleyb**
Fixes #1232

### CODE REVIEW

File: app/controllers/users/sessions_controller.rb [hunk 1]
```diff
@@ -11,5 +11,6 @@ def redirect_if_university
     service = OregonDigital::UserAttributeService.new(params[:user])
     redirect_path = service.email_redirect_path
     redirect_to redirect_path unless redirect_path.nil?
```
Comment (CGillen) [inline app/controllers/users/sessions_controller.rb:13]: Does this still redirect institutional users without admin, depositor, or collection manager roles to the dashboard?

Comment (straleyb) [reply app/controllers/users/sessions_controller.rb:13]: oh good question. ill check

Comment (straleyb) [reply app/controllers/users/sessions_controller.rb:13]: Answer: Yes, logging in via institutional login without admin depositor or collection manager still redirects to dashboard

Comment (CGillen) [reply app/controllers/users/sessions_controller.rb:13]: I think we want only those three roles to hit the dashboard, everyone else should go to the homepage or the page they left from (if easy)

Comment (straleyb) [reply app/controllers/users/sessions_controller.rb]: ahhh ok i think the title to the ticket might have misled me. Ill swing back around and verify that everyone except admin/collectionmanagers/depositors route to homepage. Else go to dashboard.

**Pull request merged**